.github/workflows/build.yml

```
@@ -0,0 +1,60 @@
+name: Build latest
+on:
+    push:
+        branches:
+            - main
+env:
+  FORCE_COLOR: true
+
+jobs:
+    Build:
+        runs-on: ubuntu-latest
+
+        steps:
+            - uses: actions/checkout@v3
+
+            - uses: pnpm/action-setup@v2 # Install pnpm using packageManager key in package.json
+
+            - name: Use Node.js 18
+              uses: actions/setup-node@v2
+              with:
+                  node-version: 18
+                  cache: "pnpm"
+
+            - name: Install dependencies
+              run: pnpm install --frozen-lockfile && sudo apt install -y wget
```

```suggestion
              run: pnpm install --frozen-lockfile
```

Unless I'm blind this doesn't seem used anymore